configs/virt/incus/scripts/buildimage.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -ex
+
+IMAGE_NAME=skiffos/testing
+CONTAINER=skiff
+OUTPUT_PATH="$BUILDROOT_DIR/output"
+IMAGES_PATH="$BUILDROOT_DIR/images"
+WORKING_PATH="${BUILDROOT_DIR}/nspawn-run"
+PERSIST_PATH="${WORKING_PATH}/nspawn-persist"
+roottar="${IMAGES_PATH}/rootfs.tar"
+
+command -v incus > /dev/null 2>&1 || {
+  echo "Failed to find the incus command. Is incus installed on your host system?"
+  exit 1
+}
+
+mkdir -p "${PERSIST_PATH}"
+TMPDIR="$(mktemp -d)"
```

Comment:
For permissions reasons can you make the tmpdir within BUILDROOT_DIR/tmp?

I believe the other scripts do this too to avoid leaking stuff into /tmp

Thanks!

Reply:
Alright! I think it will also be a good idea to drop the random folder name so we don't accidentally break reproducibility.